storage/api/Storage/Storage.cs

```diff
@@ -1243,6 +1245,46 @@ private void GetUniformBucketLevelAccess(string bucketName)
         }
         // [END storage_get_uniform_bucket_level_access]
 
+        // [START storage_enable_bucket_lifecycle_management]
+        public void EnableBucketLifecycleManagement(string bucketName)
+        {
+            var storage = StorageClient.Create();
+            var bucket = storage.GetBucket(bucketName);
+            bucket.Lifecycle = new Bucket.LifecycleData
+            {
+                Rule = new List<Bucket.LifecycleData.RuleData>
+                {
+                    new Bucket.LifecycleData.RuleData
+                    {
+                        Condition = new Bucket.LifecycleData.RuleData.ConditionData{
+                            Age = 100,
+                        },
```

Comment:
nit: either 
```suggestion
                        Condition = new Bucket.LifecycleData.RuleData.ConditionData { Age = 100, },
```

or
```suggestion
                        Condition = new Bucket.LifecycleData.RuleData.ConditionData
                        {
                            Age = 100,
                        },
```